app/presenters/two_factor_login_options_presenter.rb

```diff
@@ -28,11 +28,19 @@ def title
   end
 
   def heading
-    t('two_factor_authentication.login_options_title')
+    if reauthentication_context?
+      t('two_factor_authentication.login_options_reauthentication_title')
+    else
+      t('two_factor_authentication.login_options_title')
+    end
   end
 
   def info
-    t('two_factor_authentication.login_intro')
+    if reauthentication_context?
+      t('two_factor_authentication.login_intro_reauthentication')
+    else
+      t('two_factor_authentication.login_intro')
+    end
```

Comment:
For completeness, can we add some test coverage for this logic in `spec/presenters/two_factor_login_options_presenter_spec.rb` ?